pkg/agent/route/route_linux.go

```diff
@@ -1245,6 +1258,59 @@ func (c *Client) initWireguard() {
 	}
 }
 
+func (c *Client) initNodeLatency() {
+	gateway := c.nodeConfig.GatewayConfig.Name
+	if c.networkConfig.TrafficEncapMode.IsNetworkPolicyOnly() {
+		gateway = c.networkConfig.TransportIface
+	}
+	icmpZero := int32(0)
+
+	if c.networkConfig.IPv6Enabled {
+		antreaInputChainRulesIPv6 := []string{
+			iptables.NewRuleBuilder(antreaInputChain).
+				MatchInputInterface(gateway).
+				MatchICMP(&icmpZero, &icmpZero, iptables.ProtocolIPv6).
```

Comment:
It's not the same values for IPv6 and IPv4. See https://pkg.go.dev/golang.org/x/net/ipv6